packages/core/src/storage/migration/__tests__/__snapshots__/0.1.test.ts.snap

```diff
@@ -53,10 +53,8 @@ Object {
       },
       "id": "574b2a37-1db1-4af1-a3bf-35c6cb9e1d7a",
       "metadata": Object {
-        "_internal/indyCredential": Object {
```

Comment:
Something is not correct here. this is back to the old format and shouldn't be changed.